Update ntp_packets

d0m84 opened the pull request:
With Debian 10 the command "ntpq -c iostats -c sysstats" produces one empty line.
The additional if condition tackles this.

Traceback (most recent call last):
  File "/etc/munin/plugins/ntp_packets", line 91, in <module>
    stats[line.split(':')[0]] = int(line.split(':')[1])
IndexError: list index out of range

sumpfralle commented:
Thanks for improving this plugin!

Regarding the conditional:
* is the splitting not needed anymore with Debian Buster? (after your change it would not be executed anymore)
* could you please add a comment describing the necessity of this conditional? This will allow us to get rid of it in ten years :)

Thanks!

d0m84 commented:
>     * is the splitting not needed anymore with Debian Buster? (after your change it would not be executed anymore)

It is still needed. The condition just makes sure that lines which can't be splitted into 2 parts by delimiter : will be skipped.

>     * could you please add a comment describing the necessity of this conditional? This will allow us to get rid of it in ten years :)

The current version doesn't work with ntpq version of Debian 10. It produces the mentioned Python exception:

```
Traceback (most recent call last):
File "/etc/munin/plugins/ntp_packets", line 91, in
stats[line.split(':')[0]] = int(line.split(':')[1])
IndexError: list index out of range
```

sumpfralle commented:
Thanks for your explanation!

Could you please add a comment to the code, that explains, under which circumstances the conditional is used (and necessary) and under which circumstances it works without the conditional?

I would like to keep this plugin maintainable by allowing future contributors to understand the current code. Thank you!

sumpfralle commented:
@d0m84: ping?

d0m84 commented:
@sumpfralle done

sumpfralle commented:
Thank you!
